rllib/agents/slateq/slateq_tf_policy.py

```diff
@@ -157,21 +157,18 @@ def build_slateq_losses(
     q_clicked = tf.gather(replay_click_q, clicked_indices)
     target_clicked = tf.gather(target, clicked_indices)
 
-    def get_train_op():
-        td_error = q_clicked - target_clicked
-        if policy.config["use_huber"]:
-            loss = huber_loss(td_error, delta=policy.config["huber_threshold"])
-        else:
-            loss = tf.math.square(td_error)
-        loss = tf.reduce_mean(loss)
-        return loss, tf.reduce_mean(tf.abs(td_error))
-
-    loss, mean_td_error = tf.cond(
-        pred=tf.greater(tf.reduce_sum(input_tensor=clicked), 0),
-        true_fn=get_train_op,
-        false_fn=lambda: (tf.constant(0.0), tf.constant(0.0)),
-        name="",
+    td_error = tf.where(
+        tf.cast(clicked, tf.bool),
+        replay_click_q - target,
+        tf.zeros_like(train_batch[SampleBatch.REWARDS]),
     )
+    if policy.config["use_huber"]:
```

Comment:
oh wait whoops i totally missed this my bad

Reply:
:)